draft-barnes-acme.md

```diff
@@ -1291,110 +1291,97 @@ The server provides the following fields as part of the challenge:
 type (required, string):
 : The string "proofOfPossession"
 
-alg (required, string):
-: A token indicating the cryptographic algorithm that should be used by the client to compute the signature {{RFC7518}}.  (MAC algorithms such as "HS*" MUST NOT be used.)  The client MUST verify that this algorithm is supported for the indicated key before responding to this challenge.
-
-nonce (required, string):
-: A random 16-byte octet string, Base64-encoded
+identifier (required, identifier):
+: The ACME identifier for which authorization is being validated
 
 hints (required, object):
 : A JSON object that contains various clues for the client about what the requested key is, such that the client can find it.  Entries in the hints object may include:
 
-jwk (required, object):
-: A JSON Web Key object describing the public key whose corresponding private key should be used to generate the signature {{RFC7517}}
-
-certFingerprints (optional, array):
-: An array of certificate fingerprints, hex-encoded SHA1 hashes of DER-encoded certificates that are known to contain this key
-
-certs (optional, array):
-: An array of certificates, in Base64-encoded DER format, that contain acceptable public keys.
+  jwks (required, array of JWK):
+  : A JSON Web Key object describing acceptable public keys
 
-subjectKeyIdentifiers (optional, array):
-: An array of hex-encoded Subject Key Identifiers (SKIDs) from certificate(s) that contain the key.  Because of divergences in the way that SKIDs are calculated {{RFC5280}}, there may conceivably be more than one of these.
+  certs (optional, array of string):
+  : An array of certificates, in Base64-encoded DER format, that contain acceptable public keys.
```

Comment:
It avoids making the server transcode between certificates and JWKs, and seems pretty harmless.